redis/commands/graph/commands.py

```diff
@@ -214,6 +217,26 @@ def explain(self, query, params=None):
         plan = self.execute_command(EXPLAIN_CMD, self.name, query)
         return ExecutionPlan(plan)
 
+    def constraint(
+        self,
+        key: KeyT,
+        op: str,
+        type: str,
+        entity_type: str,
+        label_name: str,
+        n_props: int,
+        props: List[str]):
+        """
+        Constraint operation,
```

Comment:
Describe what a constraint operation is. These docs force people to go to the module docs - but we need to help the client users themselves.

Reply:
Once we will have the docs ready I'll copy the description here.